docs/iot-platform-sdk/running-the-demo.md

```diff
@@ -454,47 +454,111 @@ The above directory contains several subdirectories, each representing a device,
+device and will be renamed to the value stored in the file **'package_name'**, say 'linux64.sh'. Similarly, the file **'payload.bin'** will be downloaded to the device and will be renamed to the value stored in the file **'payload_name'**, say 'payload.bin'. Finally, the value of the file **'binsh-linux64'**, **'/bin/sh linux64.sh'**, will be executed as command in the device.
+
+```json
 [
   {
     "module": "sdo_sys",
     "msg": "filedesc",
     "valueLen": -1,
-    "valueId": "package_name",
+    "valueId": "payload_name",
     "enc": "base64"
   },
   {
     "module": "sdo_sys",
     "msg": "write",
     "valueLen": -1,
-    "valueId": "package.sh",
+    "valueId": "payload.bin",
     "enc": "base64"
   },
   {
     "module": "sdo_sys",
     "msg": "filedesc",
     "valueLen": -1,
-    "valueId": "payload_name",
+    "valueId": "package_name",
     "enc": "base64"
   },
   {
     "module": "sdo_sys",
     "msg": "write",
     "valueLen": -1,
-    "valueId": "payload.bin",
+    "valueId": "package.sh",
     "enc": "base64"
   },
   {
     "module": "sdo_sys",
     "msg": "exec",
     "valueLen": -1,
-    "valueId": "sh-linux64",
+    "valueId": "binsh-linux64",
     "enc": "base64"
   }
 ]
 ```
+
+!!! Note
+     The ordering of message blocks are important for **Client-SDK** / **Client-CSDK TPM** device. The “filedesc” block must always be followed by “write” block and if any executable script is transfered then the “exec” block must be immediately followed after both “filedesc” and “write” blocks.
+
+For the scenario of transferring multiple executables to the Device, then the following order of blocks should be adhered.
+```json
+[ {
+      "module": "sdo_sys",
+      "msg": "filedesc",
+      "valueLen": -1,
+      "valueId": "package1_name",
+      "enc": "base64"
+    },
+    {
+      "module": "sdo_sys",
+      "msg": "write",
+      "valueLen": -1,
+      "valueId": "script_one.sh",
+      "enc": "base64"
+    },
+    {
+      "module": "sdo_sys",
+      "msg": "exec",
+      "valueLen": -1,
+      "valueId": "binsh-script1",
+      "enc": "base64"
+    },
+    {
+      "module": "sdo_sys",
+      "msg": "filedesc",
+      "valueLen": -1,
+      "valueId": "package2_name",
+      "enc": "base64"
+    },
+    {
+      "module": "sdo_sys",
+      "msg": "write",
+      "valueLen": -1,
+      "valueId": "script_two.sh",
+      "enc": "base64"
+    },
+    {
+      "module": "sdo_sys",
+      "msg": "exec",
+      "valueLen": -1,
+      "valueId": "binsh-script2",
+      "enc": "base64"
+    }
+]
+```
+!!! Note
+     Using a single “exec” block you cannot execute multiple scripts on the Device.
+     For devices except Client-SDK and Client-CSDK TPM, the above order is not mandatory, we can first define “filedesc” and “write” blocks for both scripts and then define “exec” blocks later.
```

Comment:
Combine this Note with the Note present in Line 507.

Reply:
Done !